### DISCUSSION TITLE
Set a diferent php version to a specific laragon site

### DISCUSSION THREAD

**Discussion by fabiangonza** [leokhoa/laragon#792] [General]
Hi, I would like to know if there is a way to set a different PHP version to one of my Laragon sites (not to all). Right now, I am changing the PHP version to all sites using the php/version right-click menu, but it changes the PHP version for all the sites. I just want to set PHP 7.4 to one site and PHP 8.0 to the other sites. Is that possible? Maybe adding some lines to the sites-enable virtual host file? Thanks to all.

**Comment by Pen-y-Fan** [comment]
The discussion on [multi PHP #416](https://github.com/leokhoa/laragon/discussions/416) may help

**Comment by fabiangonza** [comment]
First of all thanks for your answer.
I read that post but i think is not what I need. I just want to know if
there is a way to set a specific php version to a specific app in laragon.
If I recall correctly there was a way to do that just adding a few lines in
the virtual host definition file. I preffer not to install any third party
software to do so. Once again thanks four your time and answer.
Fabián
-----------
Enviado desde dispositivo móvil

**Reply by Pen-y-Fan** [reply]
I've added another answer to the post above. Yes, Apache can be pinned to one PHP version for one site and you can switch to another PHP version in Laragon for other sites.

**Reply by fabiangonza** [reply]
Hi, I did not understand. Where did you add another answer?

**Reply by Pen-y-Fan** [reply]
See https://github.com/leokhoa/laragon/discussions/416#discussioncomment-9610740
